sdk/pom.xml

```diff
@@ -84,6 +84,12 @@
       <version>1.3.2</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.stefanbirkner</groupId>
```

Comment:
There is no need to add this dependency. All env variables can be override by Java properties in our Properties solutions in this project. That was done on purpose.